e2e/runner/sample-apps-shared/helpers/setup-environment-files.ts

```diff
@@ -0,0 +1,143 @@
+import fs from "fs";
+
+import { findUpSync } from "find-up";
+import path from "path";
+
+import { logWithPrefix } from "./log-with-prefix";
+
+type EnvFilePaths = {
+  exampleEnvFileFullPath: string;
+  exampleEnvFileName: string;
+  outputEnvFileName: string;
+  outputEnvFileFullPath: string;
+};
+
+const EXAMPLE_ENV_FILE_NAMES = [".env.example", ".env.sample"];
+
+function getEnvFilePaths({
+  rootPath,
+  installationPath,
+}: {
+  rootPath: string;
+  installationPath: string;
+}): EnvFilePaths | null {
+  const exampleEnvFileFullPath = findUpSync(EXAMPLE_ENV_FILE_NAMES, {
```

Comment:
env file may be located somewhere between rootPath and installationPath.
I used a package, but can rewrite it without using a dependency if you want.